src/styles/common.css

```diff
@@ -52,13 +52,24 @@ cite {
 }
 
 pre {
+  white-space: pre-line;
+  background-color: var(--box-colour);
+  padding: 0.5rem;
+  padding: 1em;
+
+  & code {
+    display: block;
+  }
+}
+
+/* pre {
   white-space: normal;
   background-color: var(--box-colour);
   padding: 0.5rem;
   padding: 1em;
   overflow-x: hidden;
   word-wrap: break-word;
-}
+} */

```

Comment:
Yup, looks good. You can use the var(--d) for the dimensions and delete the old version.

I was thinking we might want to style code not in `pre` blocks better. The default looks a little clunky. We could copy MDN, or even GitHub!